packages/components/pages/f-checkout/src/log-events.js

```diff
@@ -63,7 +63,7 @@ export default {
     },
     CheckoutGetFailure: {
         logMessage: 'Get Checkout Failure',
-        logMethod: 'warn'
+        logMethod: 'error'
```

Comment:
I presume this will now start paging the on-call engineers, who should they wake when this happens?

Reply:
This event belongs to an API of getting the details to fulfill an order, which is maintained by the checkout platform team.

Reply:
Where are the alerts configured? I'm still a little concerned that we'll end up getting paged for this, even if it's owned by the checkout platform team

Reply:
After discussion we agreed to keep the alert type as warn. We may revisit this again if there's a need to change it.